src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -139,6 +151,15 @@ private T getOne(Request req, Response res) {
         return getObjectForId(req, id);
     }
 
+    /**
+     * HTTP endpoint to get one entity specified by a field value.
+     */
+    private T getOneByField(Request req, Response res) {
```

Comment:
I'd like this renamed to `getFirstByFieldValue` to indicate that this is getting the first record that matches the given field value.